frontend/src/hooks/useSessionValid.js

```diff
@@ -141,10 +150,10 @@ function useSessionValid() {
       // Set the session details
       setSessionDetails(getSessionData(userAndOrgDetails));
     } catch (err) {
-      // TODO: Throw popup error message
-      // REVIEW: Add condition to check for trial period status
+      setAlertDetails(handleException(err));
       if (err.response?.status === 402) {
-        handleException(err);
+        navigate("/trial-expired");
+        return;
```

Comment:
Will this check here catchs trail expired error from other endpoints?

Reply:
@jagadeeswaran-zipstack Addressed in this comment: https://github.com/Zipstack/unstract/pull/662#discussion_r1746686818